### ISSUE TITLE
[Bug] on Windows - "greenpress start dev" and "npm run dev" cause unexpected errors

### ISSUE THREAD

**Issue by noyagasi** [greenpress/greenpress-cli#13]
**steps to reproduce:**
(After you have npm and Docker)
1. Start Installation on "Getting started with Greenpress".
2. Run npm install -g @greenpress/cli
3. Run greenpress create my-app
4. Run cd my-app
5. run greenpress start dev
or
5. run npm run dev

**You'll get these errors:**
greenpress start dev:
![greenpressstartdev](https://user-images.githubusercontent.com/25197581/90102784-089c4980-dd4a-11ea-82b4-6c8b8736424f.png)

npm run dev:
![npmrundev](https://user-images.githubusercontent.com/25197581/90102810-194cbf80-dd4a-11ea-9c12-b5ff4598b9ff.png)

**Comment by itays123** [comment]
I ran into the exact same problem on my windows 10 machine.
Any solutions so far?

additional logging output I got after running `greeenpress start dev`:
```
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:273:12)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn npm',
  path: 'npm',
  spawnargs: [ 'run', 'dev' ]
}
```

**Comment by davidmeirlevy** [comment]
@noyagasi @itays123 @drorrose 
The 3 of you together is the best match to work it out.
After I changed the configurations mechanism, it will be much easier to solve.

**Comment by itays123** [comment]
On it!

**Comment by drorrose** [comment]
The issue seems to be with the path that npm tries to locate the script at...I suggested that we try to override it with the full path, but I don't have a windows environment to test it, so @itays123 will test it and if it works we will try to locate the issue that causes it

**Comment by itays123** [comment]
I'll test it as soon as I'll get to my computer. In the meantime, some related stack overflows:

https://stackoverflow.com/questions/14381898/local-dependency-in-package-json

https://stackoverflow.com/questions/15806241/how-to-specify-local-modules-as-npm-package-dependencies?rq=1

**Comment by davidmeirlevy** [comment]
We put this issue on the greenpress-cli but it actually a bug that should be fixed at cli-service-pm2

**Comment by davidmeirlevy** [comment]
this will be fixed on greenpress/greenpress#6  issue, so i'm cloing this issue.
